src/main.js

```diff
@@ -8,6 +8,11 @@ var opts = require('nomnom')
     default: 'config.json',
     help: 'Configuration file'
   })
+  .option('bind', {
+    abbr: 'b',
+    default: "::",
```

Comment:
Please use single-quotes (`'`) to match the code style of the rest of the application.
Or even better, it might be a good idea to use `default: undefined` and let the `app.listen` do the logic:
```
If the hostname is omitted, the server will accept connections on any IPv6 address (::) when IPv6 is available, or any IPv4 address (0.0.0.0) otherwise.
```